src/shared-content/tentacle/configure-k8s-worker.include.md

```diff
@@ -0,0 +1,23 @@
+You can install the Kubernetes Worker using [Helm](https://helm.sh/) through the [octopusdeploy/kubernetes-agent](https://github.com/OctopusDeploy/helm-charts/tree/main/charts/kubernetes-agent) chart. This chart is hosted on [Dockerhub](https://hub.docker.com/r/octopusdeploy/kubernetes-agent) and can be pulled directly via the Helm CLI. 
+
+To make things easier, Octopus provides an installation wizard that generates the Helm command for you to run.
+
+:::div{.warning}
+Helm will use your current kubectl config, so make sure your kubectl config is pointing to the correct cluster before executing the following helm commands.
+You can see the current kubectl config by executing:
+```bash
+kubectl config view
+```
+:::
+
+1. In the Octopus Web Portal, navigate to the **Infrastructure** tab, select **Workers**, and click **ADD WORKER**
+2. Choose **Kubernetes** and click **ADD** on the Kubernetes Worker card.
+3. Enter a **Name** for the worker, and select the **Worker Pools** to which the worker should belong, and select **NEXT**
+   1. The dialog permits for the inline creation ofa  worker pool via the **+** button.
+   2. Click **Show advanced** to provide a custom Storage class or override the Octopus Server URL if required
+4. Select the desired language (bash or powershell), then copy and execute the supplied helm install command in a terminal configured with your k8s cluster, and click **NEXT**
```

Comment:
"Select the desired shell"

Reply:
> then copy and execute the supplied helm install command in a terminal configured with your k8s cluster

This, combined with step 5, is a bit confusing because we pretty much have the same instruction twice.

Reply:
split it up, a little.